README.md

```diff
@@ -2,6 +2,10 @@
 
 ## Working with the project
 
+### Running mongo
```

Comment:
Not saying one needs to use mongo:4.0.

I have an old mac and 4.2 didn't work :D